README.md

```diff
@@ -220,6 +222,11 @@ Web UI Options
   --ui-pass string    Web UI password. Required if --ui is specified
   --ui-runtime mins   Time the Web UI server is permitted to run. Default: 10
 
+Setup command
+
+  -o, --otc string   Setup device using a one time code
+  -u, --ff-url url   URL of FlowFuse. Required for setup
```

Comment:
Hmmmm. Yes, that does seem like a nice default but as we offer the URL in the "Device Configuration" dialog, should we also remove it from there?  It would have to be smart about that though since the users FF platform may be self hosted.

Thoughts please @knolleary before I proceed on this.

Reply:
Lets leave it as you have done. Don't want to make things more complicated.